webrtc/tools/.eslintrc.js

```diff
@@ -111,6 +114,7 @@ module.exports = {
     getUserMediaTracksAndStreams: true,
     performOffer: true,
     Resolver: true,
+    addTrackOrTransceiver: true,
```

Comment:
Ah, I did not know about that file.
Should I have updated it when adding some helper functions in RTCPeerConnection-helpers.js?

Reply:
The lint tool was intended as quick way to find out common errors such as undefined variable without being flooded by warnings on the global helpers. It has not been updated for some time so running lint now is not very useful anyway. We probably should either add all global variables to it, or remove the lint tools from the repo.

Reply:
Either we should enforce running the lint tool for every PR or we should remove it.
Do you think we should do with one or the other?